libraries/AP_InertialSensor/AP_InertialSensor_ExternalAHRS.cpp

```diff
@@ -0,0 +1,52 @@
+#include <AP_HAL/AP_HAL.h>
+#include "AP_InertialSensor_ExternalAHRS.h"
+#include <AP_ExternalAHRS/AP_ExternalAHRS.h>
+
+#if HAL_EXTERNAL_AHRS_ENABLED
+
+const extern AP_HAL::HAL& hal;
+
+AP_InertialSensor_ExternalAHRS::AP_InertialSensor_ExternalAHRS(AP_InertialSensor &imu, uint8_t _serial_port) :
+    AP_InertialSensor_Backend(imu),
+    serial_port(_serial_port)
+{
+}
+
+void AP_InertialSensor_ExternalAHRS::handle_external(const AP_ExternalAHRS::ins_data_message_t &pkt)
+{
+    Vector3f accel{pkt.accel[0], pkt.accel[1], pkt.accel[2]};
```

Comment:
There's no ID check here like there is in Baro.

I can see this is a "handle single instance for now" sort of a thing, but we should probably be consistent....

Reply:
I'll revisit when we add support for multiple external, probably when we output this from AP_Periph